### PR TITLE
VDS-333,VDS-332: Add customerSegment-list blade

### PR DESCRIPTION
**Pull request by ilyawzrd**
Demo delete segment functionality: https://virtocommerce.atlassian.net/browse/VDS-332
Demo segment-list blade: https://virtocommerce.atlassian.net/browse/VDS-333

### CODE REVIEW

**Review by trueboroda — DISMISSED** [review]

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json [hunk 1]
```diff
@@ -1,13 +1,22 @@
 {
   "demoCustomerSegmentsModule": {
     "blades": {
+      "customer-segment-list": {
+        "labels": {
+          "name": "Segment name",
+          "isActive": "Is active",
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json:7]: is-active

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json [hunk 2]
```diff
@@ -53,6 +62,12 @@
           "name": "Name"
         }
       }
+    },
+    "dialogs": {
+      "customer-segment-delete": {
+        "title": "Delete confirmation",
+        "message": "Customer {{segments}} will be deleted and the customers of this {{count}} will no longer be receiving the tagged promotions"
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json:69]: Is this message related to AC?

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Scripts/blades/customerSegment-list.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+                uiGridHelper.initialize($scope, gridOptions, function (gridApi) {
+                    //update gridApi for current grid
+                    $scope.gridApi = gridApi;
+                    uiGridHelper.bindRefreshOnSortChanged($scope);
+                    $scope.gridApi.infiniteScroll.on.needLoadMoreData($scope, showMore);
+
+                });
+
+                $timeout(function () {  blade.refresh(); });
```
Comment (pushnitsa) [inline src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Scripts/blades/customerSegment-list.js:147]: What for you using $timeout service here?

**Review by pushnitsa — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json:69]: This message is inside the modal when deleting a segment

Comment (pushnitsa) [reply src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json:69]: Just split this messages for one segment and several.

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json (hunk 1 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json]: done

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Scripts/blades/customerSegment-list.js (hunk 3 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Scripts/blades/customerSegment-list.js]: done, removed from code

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoCustomerSegmentsModule.Web/Localizations/en.VirtoCommerce.DemoCustomerSegmentsModule.json]: done

**Review by pushnitsa — APPROVED** [review]

**Review by trueboroda — APPROVED** [review]

**Comment by ilyawzrd** [comment]
Closed. All PR code moved to https://github.com/VirtoCommerce/vc-module-demo-customer-segments/pull/12
